### ISSUE TITLE
Generated Code has misordered modifiers 

### ISSUE THREAD

**Issue by hollesse** [eclipse/xtext-xtend#554]
If I use the method `addField()` from MutableClassDeclaration and want to make the field static and final these modifiers are misordered. 

Example code:
```
private def addTestField(MutableClassDeclaration class, extension TransformationContext context){
	class.addField("TEST", [
		type = boolean.newTypeReference
		final = true
		static = true
		visibility = Visibility::PRIVATE
		initializer = [ '''«false»''']
	])
}
```

Then the result is `private final static boolean TEST = false;` 
If you take a look at the java language spec it should be `private static final boolean TEST = false;`

**Comment by cdietrich** [comment]
this is a general issue that is not tight to active annotation (see XtendGenerator.xtend)

**Comment by miklossy** [comment]
We implemented the requested feature, the modifiers should be generated in the right order:  `static final` instead of `final static`.

@hollesse Would you please give it a try with the latest Xtend Version (nightly build) and give feedback, if the current behaviour corresponds to your expectations?

**Comment by kthoms** [comment]
It is not necessary to use the Nightly build. It is contained in the 2.15.0 release version.

**Comment by cdietrich** [comment]
no its not

**Comment by cdietrich** [comment]
Fixed in 2.16
